openspec/changes/flutter-login-and-app-shell/specs/flutter-auth-shell/spec.md

```
@@ -0,0 +1,68 @@
+<!-- SPDX-License-Identifier: AGPL-3.0 -->
+<!-- Copyright (C) 2024-2026 Breakdown RS Contributors -->
+<!-- Co-authored-by: glm-5.3 (neuralwatt) -->
+
+# flutter-auth-shell Specification (delta)
+
+## ADDED Requirements
+
+### Requirement: Root Auth Gate on Application Entry
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)